src/app/resources/browser/assets/css/darkmode.css

```diff
@@ -1,11 +1,28 @@
 /* Dark mode (see #466). */
 /* TODO: Check if we need -webkit-filter for old Qt WebKit. */
+
+body {
+    background-color: white !important;
+    transition-duration: 0s !important;
+}
+
 html {
-    -webkit-filter: invert() hue-rotate(180deg) contrast(70%);
-    filter: invert() hue-rotate(180deg) contrast(70%);
+    background-color: #262626 !important;
+    filter: invert(1) hue-rotate(180deg) contrast(70%) !important;
 }
 
 html img {
-    -webkit-filter: invert() hue-rotate(180deg) contrast(120%);
-    filter: invert() hue-rotate(180deg) contrast(120%);
+    filter: invert(1) hue-rotate(180deg) contrast(120%) !important;
+}
+
+::-webkit-scrollbar {
```

Comment:
I think scroll bars should follow the system theme not web page, will remove this.

Reply:
The problem with this was, in dark mode I have a complete dark window but with bright scrollbars in the docset part. The WebEngine don't use the the scrollbar style from the Qt Application.

Reply:
I see. OK, let's keep this for now. In general I think the dark mode should be smarter and follow application's theme.

Reply:
Unfortunately, it's a bit broken, no buttons. I'll look into styling scroll bars later.

Reply:
![image](https://user-images.githubusercontent.com/714940/80322017-e4931780-87ef-11ea-8e28-2c7b41973562.png)

Reply:
I have always steadfastly refused to use ::-webkit-scrollbar styling because it mangles the scrollbar in this way. Hopefully they’ll follow Firefox’s lead soon, in implementing scrollbar-color and the rest of that lot, *and* in defaulting to a dark scrollbar (without ruining the controls) on dark content.